backend/controller/AccountController.ts

```diff
@@ -135,16 +135,19 @@ export class AccountController {
     public static async ensureAdminAccount(): Promise<void> {
         const user = await DBUser.findOneBy({ type: UserClass.ADMIN });
         if (user === null) {
-            const adminUser: Partial<IUser> = {};
+            const adminUser = new DBUser();
+            adminUser.type = UserClass.ADMIN;
             adminUser.name = 'admin';
             adminUser.mail = 'admin@loernwerk.de';
             const pw = crypto.randomBytes(16).toString('hex');
             console.log(
                 'Admin account created, username: admin, mail: admin@loernwerk.de, password: ' +
```

Comment:
Could we insert username and email from the DBUser?

Reply:
you mean from a config file?

Reply:
I assume he means replacing them inside the `console.log` instead of hardcoding them - might be an idea, so you could change it in one place instead of having two places - I'd also put the logging message below the saving, so if the log message gets printed, the account has already been created, which is currently indicated by the message

Reply:
Yes, ment that

Reply:
done

Reply:
Do we want to leave this as concatination or convert to the insertion method used in most other strings in the project

Reply:
you mean like this example: `var s = "hello ${my_name}, how are you doing";`?

Reply:
yes

Reply:
I'd say so, but with backticks ` instead of " like
```js
console.log(`something${someVariable}`)
```
since that's how JS does it

Reply:
kk

Reply:
alright done